dbt_af/operators/run.py

```diff
@@ -72,6 +69,10 @@ class DbtSnapshot(DbtBaseDatasetOperator):
     def cli_command(self) -> str:
         return 'snapshot'
 
+    def __init__(self, dbt_af_config: 'Config', **kwargs) -> None:
+        self.retry_policy = dbt_af_config.retries_config.dbt_snapshot_retry_policy
```

Comment:
Can't we move this logic to base class?

Reply:
Each operation resolves it's own retry policy based on it's type